app/helpers/dashboard_helper.rb

```diff
@@ -40,6 +40,10 @@ def future_distributed
     number_with_delimiter future_distributed_unformatted
   end
 
+  def recently_added_user_display_text(user)
+    (user.name.presence == "Name Not Provided") ? user.email : user.name
```

Comment:
You don't need `presence` here because `nil` will by definition not equal the given string :) You also don't need the additional parentheses.

```ruby
user.name == 'Name Not Provided' ? user.email : user.name
```

Reply:
> You don't need `presence` here because `nil` will by definition not equal the given string :) You also don't need the additional parentheses.
> 
> ```ruby
> user.name == 'Name Not Provided' ? user.email : user.name
> ```

It seems like the lint is failing without the parentheses